webview/src/experiments/components/table/styles.module.scss

```diff
@@ -1,7 +1,7 @@
 // Variables
 
 @import '../../../shared/variables.scss';
-@import '../../../shared/style.scss';
+@import '../../../shared/styles.scss';
```

Comment:
I don't think this line is necessary as it is already imported in `webview/src/experiments/index.tsx`. There doesn't seem to be any variables or classes to extend in there.

Reply:
Looks like we are extending an `a` element, which should be done in a mixin... I'll go ahead and create a `shared/mixins.scss` file similar to our `variables.scss` file.